tldw_Server_API/app/core/MCP_unified/command_runtime/presentation.py

```
@@ -0,0 +1,437 @@
+    return 4_000
+
+
+def _resolve_spill_root(spill_dir: Path | str | None) -> Path:
+    if spill_dir is not None:
+        spill_root = Path(spill_dir)
+    else:
+        spill_root = Path(tempfile.gettempdir()) / f"mcp-command-presentation-{_default_spill_suffix()}"
+
+    if spill_root.exists():
+        if spill_root.is_symlink():
+            raise PermissionError(f"Refusing to use symlink spill directory: {spill_root}")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)